services/README.md

```diff
@@ -1,23 +1,55 @@
 # services
 
-Provide access to services from docker registry
+Each folder contains a services that is part or can be spawned by the platform.
+The prefix *dy-* in the naming indicates that this service is not a building
+part of the application (i.e. not listed as an services in the docker-compose file)
+but instead it is *dy*namically spawned by the director as a back-end service.
```

Comment:
computation folder should be then prefixed isn't it?

Reply:
Computations folder still empty .. I would wait to @mguidon takes the decision